### DISCUSSION TITLE
A keybinding to toggle the dock?

### DISCUSSION THREAD

**Discussion by charlesetc** [zed-industries/zed#6475] [Feature Requests]
Hi! 

It seems as though there isn't a keybinding to toggle the dock, but instead different ones to pin the dock to places and then one to hide it. I'd like to have one that toggles it basically, is that a thing?

If not, is there a way to check whether the dock is currently hidden in the context of keybindings? I'm imagining this might work in the same way one can check the current vim mode:

```
    {
        "context": "Editor && vim_mode == visual",
        "bindings": { ...
```

Thanks in advance!

**Comment by JosephTLyons** [comment]
Hey @charlesetc, try `shift-esc`.

**Reply by charlesetc** [reply]
That seems to toggle whether a particular tab is in the Dock. I'm hoping to toggle whether the Dock is open at all. Any ideas?

**Reply by JosephTLyons** [reply]
> That seems to toggle whether a particular tab is in the Dock. I'm hoping to toggle whether the Dock is open at all. Any ideas?

Sorry about that, I put the wrong command in.  I edited the comment above.  The problem here is, the dock button doesn't have a tooltip on it showing the key combo and for some reason, it isn't showing in the command palette next to the action, so the key combo is totally undiscoverable.

**Reply by charlesetc** [reply]
Ah true - perfect thank you!
